species5.json

```diff
@@ -1741,8 +1741,8 @@
     "LEAF_COLOR": "blue,green,orange,yellow",
     "LEAF_ARRANGEMENT": "",
     "LEAF_UNDERSIDE_COLOR": "",
-    "LEAF_TEXTURE": "",
-    "DESCRIPTION": "Blue Oaks are medium-sized deciduous trees growing up to 50-82 feet (15.24-24.99 meters) tall, usually with a somewhat irregularly-shaped crown, and a trunk 1.5-3 feet (.45-.91 meters) in diameter. This species of oak is endemic to California, and is commonly found in the Coast Ranges and the foothills of the Sierra Nevada. It is California's most drought-tolerant deciduous oak. The trees grow slowly, about 12 inches (30 cm) per year. Individual trees over 500 years old have been recorded. The blue-green leaves are tough and leathery - and either have smooth margins or are extremely shallowly lobed. As of 2002, it appears that the Quercus douglasii (and many other members of the white oak group) are resistant to the disease known as 'sudden oak death'. Often a blue oak tree will have galls; called “oak apples� The oak apples are a greyish-red when growing and greyish-black when dead. Immature wasps, inserted into the tree by their mother, cause them. Another kind of gall grows on the leaves, but these are smaller, usually just a spot on the leaf.",
+    "LEAF_TEXTURE": "leathery",
+    "DESCRIPTION": "Blue Oaks are medium-sized deciduous trees growing up to 50-82 feet (15.24-24.99 meters) tall, usually with a somewhat irregularly-shaped crown, and a trunk 1.5-3 feet (.45-.91 meters) in diameter. This species of oak is endemic to California, and is commonly found in the Coast Ranges and the foothills of the Sierra Nevada. It is California's most drought-tolerant deciduous oak. The trees grow slowly, about 12 inches (30 cm) per year. Individual trees over 500 years old have been recorded. The blue-green leaves are tough and leathery - and either have smooth margins or are extremely shallowly lobed. As of 2002, it appears that the Quercus douglasii (and many other members of the white oak group) are resistant to the disease known as 'sudden oak death'. Often a 'Blue Oak' tree will have galls; called “oak apples�. The oak apples are a greyish-red when growing and greyish-black when dead. Immature wasps, inserted into the tree by their mother, cause them. Another kind of gall grows on the leaves, but these are smaller, usually just a spot on the leaf.",
```

Comment:
@christyheaton so weird...so it fixed some of the issues, but 'oak apples' in the desc is still borked.